modules/neo4j/src/test/java/org/testcontainers/containers/Neo4jTestImages.java

```diff
@@ -3,5 +3,5 @@
 import org.testcontainers.utility.DockerImageName;
 
 public interface Neo4jTestImages {
```

Comment:
Should we remove this file, if most places in the test already specify the used image directly?

Reply:
@rnorth Did this extraction in the context of the deprecation of the non-args constructors. I could live with a static variable. Commit incoming.

Reply:
Maybe not because it is used in both test classes. So a central test version defining class/interface does not provide any disadvantages over two version Strings, IMHO. I do not want to have to maintain the version String twice or make one test class depend on the other one.